migtests/tests/pg/pgtbrus/expected_files/expectedAssessmentReport.json

```diff
@@ -140,6 +140,11 @@
 			"Objects": [],
 		    "MinimumVersionsFixedIn": null
         },
+		{
```

Comment:
Won't the datatype issues go under Unsupported DataTypes section? Why do we need a new feature?

Reply:
This is for `lo functions` case where it can be in any DDL as well so need a Feature as well

Reply:
Ah got it! Yeah, let's rename the feature name to `Large Object Functions`, got confused. 
So, to confirm, if it's the lo data type that is not supported, it will go into the "Unsupported Datatypes" section, right?

Reply:
yes